csbuild/_build/__init__.py

```diff
@@ -630,14 +631,38 @@ def Run():
 		else:
 			log.Error("csbuild cannot be run from the interactive console.")
 			system.Exit(1)
+			
+		from ..tools.common.tool_traits import HasOptimizationLevel
+		OptimizationLevel = HasOptimizationLevel.OptimizationLevel
+		from ..tools.common.tool_traits import HasDebugLevel
+		DebugLevel = HasDebugLevel.DebugLevel
+		#Create the default targets...
+		with csbuild.Target("release"):
+			csbuild.SetOptimizationLevel(OptimizationLevel.Max)
+			csbuild.SetDebugLevel(DebugLevel.Disabled)
+			csbuild.SetDebugRuntime(False)
+			csbuild.AddDefines("NDEBUG")
+
+		with csbuild.Target("debug"):
+			csbuild.SetOptimizationLevel(OptimizationLevel.Max)
```

Comment:
Should the debug target really be getting max optimization?

Reply:
no it should not.